main.py

```diff
@@ -5,8 +5,11 @@
 import pandas as pd
 import numpy as np
 import scipy as cp
+from pre_run import Settings
```

Comment:
better to do import pre_run
and then when you want to use the Setting class do pre_run.Settings

Reply:
Im not sure, this is the only class in this module.

Reply:
technically you can use from import. I will bring up 2 reasons for using **import** as a default vs **from import** 
1. using **from import** it can lead to naming conflicts, when 2 modules have the same names
2. using import makes the code more readable and easy to maintain since it is very clear from which module the class was brought

for example, when we use a pandas dataframe (also in our code) it is:
`import pandas
...
df = pd.DataFrame(ticker_data, index=tickers)`

Reply:
Done